packages/commonwealth/server/routes/tokenBalance.ts

```diff
@@ -45,11 +47,27 @@ const tokenBalance = async (
     throw new AppError(err)
   }
 
+  let chain_node_id = chain.ChainNode.id;
+  if (chain.ChainNode.chain_base !== 'cosmos') {
```

Comment:
We should make this check positive (i.e. only look for contract in certain circumstances) rather than negative (i.e. always look except in certain circumstances).